### PR TITLE
Language tweaks, lift-ups and fixes (mostly lore related)

### PR DESCRIPTION
**Pull request by Draganfrukts**
## About The Pull Request

Edits some languages to fit with established server lore.

## How This Contributes To The Skyrat Roleplay Experience

It makes the universe consistent.

## Changelog

:cl:
spellcheck: Pan Slavic description changed to match the lore.
tweak: Replaced "Selenian" with "Sol Common", use ",sol" to speak. The directory is the same, so you don't need to re-select it.
spellcheck: Dwarf no longer has an odd chance to drop a syllable with a percentage symbol in it.
imageadd: Sol Common language icon, replaced red star with proposed NRI flag (thanks Russian community), gave the Skrell icon a little uplifting

/:cl:

## Sprite comparisons

image

### CODE REVIEW

File: modular_skyrat/modules/customization/modules/language/selenian.dm [hunk 1]
```diff
@@ -1,16 +1,23 @@
...
+	space_chance = 60
 	syllables = list(
 		"al", "an", "ar", "as", "at", "ed", "er", "ha", "he", "hi", "is", "le", "me", "on", "se", "ti",
-		"ve", "wa", "are", "ent", "for", "had", "hat", "hin", "ch", "be", "abe", "die", "sch", "aus",
+		"ve", "wa", "ameno", "are", "ent", "for", "had", "hat", "hin", "ch", "be", "abe", "die", "sch", "aus",
 		"ber", "che", "que", "ait", "men", "ave", "con", "com", "eta", "eur", "est", "ing", "ver", "was",
-		"hin", "deed", "accord", "caveat", "pledge", "rights", "et", "ex", "bono", "quo", "pro", "ad",
-		"ek", "aur", "ki", "ki", "ke", "de", "ki", "ne", "ek", "aar", "ain", "ki", "me", "dei", "dekh"
+		"hin", "deed", "sed", "ut", "unde", "omnis", "latire", "iste", "natus", "sit", "vol", "totam", "rem", "eaque", 
+		"ipsa", "quae", "ab", "illo", "et", "quasi", "dicta", "dorime," "sunt", "enim", "ipsam", "aut", "odit", "qui",
```
Comment (Draganfrukts) [inline modular_skyrat/modules/customization/modules/language/selenian.dm]:
```suggestion
		"ipsa", "quae", "ab", "illo", "et", "quasi", "dicta", "dorime", "sunt", "enim", "ipsam", "aut", "odit", "qui", 
```
fucking idiot dragonfruits

**Comment by Draganfrukts** [comment]
I'm the best coder to have ever lived.

Pending host approval

**Comment by Stalkeros2** [comment]
Considering all the new lore, I think it'd be better to redesign NRI's flag due to them not being like what I thought they're (that is, some le monarchy).
brb

**Comment by theselfish** [comment]
Putting aside personal views on the lore (I have mixed feeling on there being a giant White Russian empire out there.);

I think making the Panslavic sprite a uhh...

Flag *VERY* similar to one that's commonly used by Russian Neo-Nazis is a bit...Bad.

**Comment by Draganfrukts** [comment]
It's just a silly little flag I shouldn't need to care who associates with it in present day 2022 it'll be completely shifted 500 years from now 

and with this I ask for nothing more related to politics because I don't want a pride pin pr comment section situation

**Comment by Lukaster42** [comment]
> 

Did you ever study history in school?

**Comment by Lukaster42** [comment]
> Putting aside personal views on the lore (I have mixed feeling on there being a giant White Russian empire out there.);
> 
> I think making the Panslavic sprite a uhh...
> 
> Flag _VERY_ similar to one that's commonly used by Russian Neo-Nazis is a bit...Bad.

Did you ever study history in school?

**Comment by Stalkeros2** [comment]
~~Man come on just ignore them for the love of god.~~

**Comment by Lukaster42** [comment]
Eh, sure.

**Comment by theselfish** [comment]
> > Putting aside personal views on the lore (I have mixed feeling on there being a giant White Russian empire out there.);
> > I think making the Panslavic sprite a uhh...
> > Flag _VERY_ similar to one that's commonly used by Russian Neo-Nazis is a bit...Bad.
> 
> Did you ever study history in school?

I did. I know it's the Imperial Russian flag.
I also know that groups like the Russian All-People's Union, Russian All-National Union and Union of the Russian People all use it. As well as other fascist groups in Russia.

**Review by Gandalf2k15 — APPROVED** [review]

**Pull request merged**